components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.ts

```diff
@@ -261,12 +263,27 @@ export class ProfileOverviewComponent implements OnInit, OnDestroy {
     });
     observableForkJoin(installProfiles).pipe(
       takeUntil(this.isDestroyed))
-      .subscribe(
-        () => this.refreshProfiles(),
-        () => this.showDownloadError()
+      .subscribe(data => {
+        if (data) {
+          this.refreshProfiles();
+          this.showDownloadError();
+        }
+      },
+      (error) => this.showAuthError(error)
       );
   }
 
+  showAuthError(error) {
+    if (error.status === HttpStatus.FORBIDDEN) {
+      this.authErrorVisible = true;
+      this.refreshProfiles();
+    }
```

Comment:
If I understand this correctly, an error *not related to auth* will be ignored here?